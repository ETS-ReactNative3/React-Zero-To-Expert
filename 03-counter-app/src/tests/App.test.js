import React from 'react'
import { render, screen } from '@testing-library/react'
import App from '../App'

describe('Pruebas en <App />', () => {
  test('debe de mostrar el mensaje "Hola, soy Goku', () => {
    const saludo = 'Hola, soy Goku'

    // const wrapper = render(<App saludo={saludo} />)
    // const { getByText } = render(<App saludo={saludo} />)
    render(<App saludo={saludo} />)

    expect(screen.getByText(saludo)).toBeInTheDocument()
    // expect(screen.getByText('Hola, soy Goku')).toBeInTheDocument()
    // expect(screen.getByText((saludo = ''))).toBeInTheDocument() // error
  })
})
