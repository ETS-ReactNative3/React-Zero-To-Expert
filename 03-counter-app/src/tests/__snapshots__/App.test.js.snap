// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <App /> debe de mostrar <App /> correctamente  1`] = `
<Fragment>
  <h1>
    Hola, soy Goku
    !!!
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
